test/test_macos_archs.py

```diff
@@ -8,10 +8,9 @@
 
 basic_project = test_projects.new_c_project()
 
-ALL_MACOS_WHEELS = (
-    utils.expected_wheels('spam', '0.1.0', machine_arch='x86_64')
-    + utils.expected_wheels('spam', '0.1.0', machine_arch='arm64')
-)
+ALL_MACOS_WHEELS = utils.expected_wheels(
+    'spam', '0.1.0', machine_arch='x86_64'
+) + utils.expected_wheels('spam', '0.1.0', machine_arch='arm64')
```

Comment:
Sure, use the proper Python 3 syntax instead of this weird construction. `{*a, *b}` - as a side benefit, it can now be a set! (Probably this should return a set in the first place...)